ivy/functional/frontends/jax/numpy/mathematical_functions.py

```diff
@@ -708,6 +709,11 @@ def trace(a, offset=0, axis1=0, axis2=1, out=None):
     return ivy.trace(a, offset=offset, axis1=axis1, axis2=axis2, out=out)
 
 
+@to_ivy_arrays_and_back
+def transpose(a, axes=None):
+    return mp_routines.transpose_like_operations.transpose(a, axes=axes)
```

Comment:
I think we should use ivy.transpose instead here

Reply:
Ah sorry, use ivy.permute_dims

Reply:
Sure, Updated the code with ivy.permute_dims